packages/hub/src/graphql/queries/runSquiggle.ts

```diff
@@ -21,6 +25,20 @@ export const squiggleValueToJSON = (value: SqValue) => {
       if (value instanceof Map) {
         return Object.fromEntries(value.entries());
       }
+      if (value instanceof SqLambda) {
+        return value.toString();
+      }
```

Comment:
todo: Also do this for calculators, and if there are any other responses that have Lambdas